jenkins_pipelines/environments/common/pipeline-reference.groovy

```diff
@@ -0,0 +1,44 @@
+def run(params) {
+    timestamps {
+        deployed = false
+        env.resultdir = "${WORKSPACE}/results"
+        env.resultdirbuild = "${resultdir}/${BUILD_NUMBER}"
+        env.common_params = "--outputdir ${resultdir} --tf ${params.tf_file} --gitfolder ${resultdir}/sumaform"
+        try {
+            stage('Clone sumaform and susemanager-ci') {
+                // Create a directory for all the outputs belonging to this build number
+                sh "mkdir -p ${resultdir}"
+                git url: params.terracumber_gitrepo, branch: params.terracumber_ref
+                dir("susemanager-ci") {
+                    checkout scm
+                }
+                // Create a directory for  to place the directory with the build results (if it does not exist)
+                sh "mkdir -p ${resultdir}"
```

Comment:
Why not use `dir` method as you did above? It will create the directory for you if it doesn't exist.

Reply:
Does it work even if I don't do anything else inside, such as `checkout scm`?

If so, I agree, I will use it.

Reply:
Yes, it does, but why not add your line bellow inside that scope?
It makes total sense, as the command needs that directory, isn't it?

Reply:
Done, an inf fact this was duplicated (done also at line 10).